moveit_ros/planning_interface/move_group_interface/src/wrap_python_move_group.cpp

```diff
@@ -123,8 +123,10 @@ class MoveGroupInterfaceWrapper : protected py_bindings_tools::ROScppInitializer
   {
     const robot_state::RobotState& values = moveit::planning_interface::MoveGroupInterface::getJointValueTarget();
     bp::list l;
-    for (const double *it = values.getVariablePositions(), *end = it + getVariableCount(); it != end; ++it)
-      l.append(*it);
+    std::vector<std::string> names = getActiveJoints();
+    for(std::vector<std::string>::const_iterator it = names.begin(); it != names.end(); ++it) {
+        l.append(*values.getJointPositions(*it));
```

Comment:
This will only return the first variable of each joint and thus not can handle multi-dof joints!